x-pack/plugins/apm/public/components/shared/KueryBar/index.tsx

```diff
@@ -48,11 +54,27 @@ export function KueryBar() {
   });
   const { urlParams } = useUrlParams();
   const location = useLocation();
+  const matchedRoutes = useMatchedRoutes();
+
   const apmIndexPatternTitle = chrome.getInjected('apmIndexPatternTitle');
   const indexPatternMissing =
     !state.isLoadingIndexPattern && !state.indexPattern;
   let currentRequestCheck;
 
+  let queryExample;
+
+  if (find(matchedRoutes, { name: TRANSACTIONS })) {
+    queryExample = 'transaction.duration.us > 300000';
+  } else if (find(matchedRoutes, { name: ERRORS })) {
+    queryExample = 'http.response.status_code >= 400';
+  } else if (find(matchedRoutes, { name: METRICS })) {
+    queryExample = 'process.pid = "1234"';
+  } else {
+    throw new Error(
```

Comment:
throwing an error ensures we don't add a view without making sure we have an appropriate query example.